doc/source/api/solver/fielddata.rst

```diff
@@ -21,6 +24,17 @@ available (for example, by reading case and data files):
   >>> solver.file.read(file_type="case", file_name=import_filename)
   >>> solver.solution.initialization.hybrid_initialize()
 
+2. Reading case and data files
+
+.. code-block:: python
+
+  >>> import ansys.fluent.core as pyfluent
+  >>> from ansys.fluent.core import examples
+  >>> import_filename = examples.download_file("mixing_elbow.msh.h5", "pyfluent/mixing_elbow")
+  >>> data_filename = examples.download_file("mixing_elbow.dat.h5", "pyfluent/mixing_elbow")
+  >>> solver = pyfluent.launch_fluent(mode="solver")
+  >>> solver.file.read(file_type="case", file_name=import_filename)
+  >>> solver.file.read(file_type="data", file_name=data_filename)
```

Comment:
@hpohekar Now that this code has changed, it has become inconsistent with the changed comments.

Reply:
@seanpearsonuk This is not clear to me.

Reply:
@hpohekar the above comment now reads "...(for example, either by reading a case file and then initializing as in the following code, ...)" which contradicts the "following code", which reading data instead.

Reply:
Do we just want "(for example, either by reading a case file and then initializing as in the following code, or by reading case and data files)"  with previous version ? No need to add reading data file code ?

Reply:
Done.